ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiActiveContractsService.scala

```diff
@@ -75,12 +71,11 @@ private[apiserver] object ApiActiveContractsService {
       executionContext: ExecutionContext,
       loggingContext: LoggingContext,
   ): ActiveContractsService with GrpcApiService = {
-    val errorFactories = ErrorFactories()
-    val field = FieldValidations(ErrorFactories())
+    val field = FieldValidations()
     val service = new ApiActiveContractsService(
       backend = backend,
       metrics = metrics,
-      transactionFilterValidator = new TransactionFilterValidator(errorFactories),
+      transactionFilterValidator = new TransactionFilterValidator(),
```

Comment:
This one as well can be made into an object

Reply:
Done